### PR TITLE
Run special-purpose pipelines less often

### PR DESCRIPTION
**Pull request by markus-hinsche**
**Problem** The zscore pipeline and test/qa pipeline run for every PR. This is not necessary and complicates unrelated PRs.

**Changes:**
- diable zscore pipeline and test/qa pipeline for PRs
- fix indent for test/qa pipeline for PRs

Notice that this PR only has 3 checks instead of 5 checks.

### CODE REVIEW

**Review by prajurock — APPROVED** [review]
Nice One.

File: src/common/eval/QA/test-pipeline.yml [hunk 1]
```diff
@@ -1,80 +1,82 @@
-  trigger:
-    batch: true
-    branches:
-      include:
-      - test-pipeline
+trigger:
+  branches:
+    include:
+    - master
```
Comment (gangooteli) [inline src/common/eval/QA/test-pipeline.yml:4]: Whenever there is change in master, this will trigger. Right ? Do you intent to do that ?

Comment (markus-hinsche) [reply src/common/eval/QA/test-pipeline.yml:4]: I am not sure what is best. @gangooteli You wrote this pipeline so let me know when it should be triggered

Comment (markus-hinsche) [reply src/common/eval/QA/test-pipeline.yml:4]: Will merge for now to improve other PRs, feel free to make a follow-up PR to adjust the trigger @gangooteli

**Pull request merged**